app/controllers/meals_controller.rb

```diff
@@ -0,0 +1,10 @@
+class MealsController < ApplicationController
+  include MealsHelper
+  def create
+    @meal = Meal.new(meal_params)
```

Comment:
Controller shouldn't have any business logic (in this case - creating meal)
[source](http://blog.codeclimate.com/blog/2012/10/17/7-ways-to-decompose-fat-activerecord-models/).